sdk-setup/src/mb2

```diff
@@ -303,8 +303,9 @@ GLOBAL OPTIONS
 KNOWN ISSUES
 
 Build target snapshot is not reset automatically when the original target is
-changed just by removing packages. Package installation or update is needed to
-trigger the automatic reset.
+changed just by removing packages via plain RPM usage. Either use zypper for
+all package operations or use 'mb2 installdeps --reset' to ensure snapshot is
```

Comment:
The UI has changed recently, please update instructions to use `mb2 build-requires reset` instead.

Reply:
Added https://github.com/sailfishos/sdk-setup/pull/114/commits/9ef37f8ff8904f65b0101d85cd9f5d9491f663bc